autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -86,18 +86,23 @@ def __init__(
                 logger.warning(
                     "overwrite_instructions is False. Provided instructions will be used without permanently modifying the assistant in the API."
                 )
-
         super().__init__(
             name=name,
             system_message=instructions,
-            human_input_mode="NEVER",
             llm_config=llm_config,
+            **kwargs
         )
-
+        self.cancellation_requested = False
```

Comment:
Is there an useage example for this argument?

Reply:
It was assumed an external object may want to cancel because the run is looping in the state machine, this is a way to try to cancel before it finishes, wasting tokens.

Reply:
Ok, Implementing a timeout mechanism for the GPT Assistant Agent would be even better.